app/services/claims/update_school_emails.rb

```diff
@@ -0,0 +1,35 @@
+require "csv"
+
+class Claims::UpdateSchoolEmails
+  include ServicePattern
+
+  def initialize(csv_string:)
+    @csv_string = csv_string
+  end
+
+  def call
+    ApplicationRecord.transaction do
+      CSV.parse(csv_string, headers: true) do |row|
+        school = School.find_by(urn: row["urn"])
+
+        if school.nil?
+          Rails.logger.info "School not found with URN: #{row["urn"]} (#{row["name"]})"
+          next
+        end
+
+        email = row["email"]
+
+        unless email =~ /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
Is there a reason to not use `URI::MailTo::EMAIL_REGEXP`?

Reply:
Also, it seems like we should add the email validation to the `School` model, no?